src/header/parsing.rs

```diff
@@ -130,10 +134,27 @@ pub fn parse_extended_value(val: &str) -> ::Result<ExtendedValue> {
     })
 }
 
+impl Display for ExtendedValue {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        let encoded_value =
+            percent_encoding::percent_encode(&self.value[..],
+                                             percent_encoding::HTTP_VALUE_ENCODE_SET);
+        if self.language_tag.is_some() {
+            write!(f,
+                   "{}'{}'{}",
+                   self.charset,
+                   self.language_tag.clone().expect("No language tag found"),
```

Comment:
We don't need to clone the language tag here (and its Strings and copy them), we can just pass a reference to the write macro.